### ISSUE TITLE
Error when identifying old Internet Explorer version

### ISSUE THREAD

**Issue by roopemerikukka** [rodneyrehm/viewport-units-buggyfill#40]
Hi,

I'm getting error when the test checks for old IE version:

```
/*@cc_on

  @if (@_jscript_version <= 10)
    isOldInternetExplorer = true;
  @end

@*/
```

The error I'm getting is: ```Unable to get property 'init' of undefined or null reference```

This can be fixed by putting variable identifier in front of ``` isOldInternetExplorer ``` So the working check would be looking like this:

```
/*@cc_on

  @if (@_jscript_version <= 10)
    var isOldInternetExplorer = true;
  @end

@*/
```

**Comment by rodneyrehm** [comment]
weird. @zoltan-dulac has more insight into Internet Explorer's Conditional Comments… *ping*

**Comment by roopemerikukka** [comment]
After giving more look for the code, I don't think this fixes any problems but only removes the error because the variable is introduced before.

I'm having the error above with IE10 / Win7 and that also stops loading of all the scripts left.

**Comment by zoltan-dulac** [comment]
@roopemerikukka Thanks so much for the bug report. Curious - I have never seen this bug before and can't replicate the issue ... I would love to know under what circumstances you are getting the original message. Can you give the following info?

- What exact version of IE are you using (you mention IE10/Win 7, but I want to know if you have seen it in others)
- Is the browser in compatability mode or standards mode? What are the exact settings?
- Do you have a URL in which we can replicate the problem?

All of that info (especially the URL) will help me troubleshoot this.

**Comment by roopemerikukka** [comment]
@zoltan-dulac I'm using Windows 7 and Internet Explorer 10.0.9200.16721. This setup is running in VirtualBox VM from https://www.modern.ie/en-us without any updates installed.

I cannot give the URL because the site we are developing is running in internal test server. I tried to create duplicate of the page but couldn't replicate the problem. Don't know if the problem is caused by combination of different js libs.

**Comment by rodneyrehm** [comment]
should've been resolved since #43 landed
